### ISSUE TITLE
Celeryworker not started after docker-compose up

### ISSUE THREAD

**Issue by semernyakov** [fastapi/full-stack-fastapi-template#208]
celeryworker_1  |     return self._get_backend()
celeryworker_1  |   File "/usr/local/lib/python3.7/site-packages/celery/app/base.py", line 949, in _get_backend
celeryworker_1  |     self.loader)
celeryworker_1  |   File "/usr/local/lib/python3.7/site-packages/celery/app/backends.py", line 74, in by_url
celeryworker_1  |     return by_name(backend, loader), url
celeryworker_1  |   File "/usr/local/lib/python3.7/site-packages/celery/app/backends.py", line 54, in by_name
celeryworker_1  |     cls = symbol_by_name(backend, aliases)
celeryworker_1  |   File "/usr/local/lib/python3.7/site-packages/kombu/utils/imports.py", line 57, in symbol_by_name
celeryworker_1  |     module = imp(module_name, package=package, **kwargs)
celeryworker_1  |   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
celeryworker_1  |     return _bootstrap._gcd_import(name[level:], package, level)
celeryworker_1  |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
celeryworker_1  |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
celeryworker_1  |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
celeryworker_1  |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
celeryworker_1  |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
celeryworker_1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
celeryworker_1  |   File "/usr/local/lib/python3.7/site-packages/celery/backends/base.py", line 10, in <module>
celeryworker_1  |     from future.utils import raise_with_traceback
celeryworker_1  | ModuleNotFoundError: No module named 'future'

**Comment by tomwojcik** [comment]
Yes, so I'm digging into that since yesterday. I have a very similar stacktrace.
The issue is celery. They... released a broken version with missing dependency.

See https://github.com/celery/celery/issues/6145

Maybe it would be smarter not to "^..." all the dependencies? I understand the tradeoff but better to be safe than sorry.

@beatum the solution would be to go to `pyproject.toml` and change

```
celery = "^4.4.2"
```

to

```
celery = "4.4.3"
```

**Comment by tomwojcik** [comment]
Even though I can confirm that setting celery version to 4.4.3 fixes the issue I wouldn't close it yet because celery team didn't release the fix yet.

**Comment by YuriiMotov** [comment]
Outdated
